### PR TITLE
allow wildcard path on taskMinify

### PR DESCRIPTION
**Pull request by kodok-buncit**
this PR will allow doing task like this.

```
$this->taskMinify( 'public/css/*.css' )
     	->run();
```

### CODE REVIEW

File: src/Task/Assets/Minify.php [hunk 1]
```diff
@@ -49,8 +53,42 @@ class Minify extends BaseTask
...
+            }
+            
+            foreach ($iterator as $file) {
+                $this->files[] = $file->getRealpath();
+            }
+
+            return $this;
         }
 
         return $this->fromText($input);
```
Comment (boedah) [inline src/Task/Assets/Minify.php]: this is never reached and breaks existing behavior...

**Comment by kodok-buncit** [comment]
@boedah  how about new commit?

**Comment by burzum** [comment]
I'm not against the feature but it seems useless to me because usually the order of the files matters. I ran into the same issue when I tried to process an angular application that has a lot sub-folders.

**Comment by boedah** [comment]
agree.
Also seems the Result object of the `_run()` method is never used...

**Comment by kodok-buncit** [comment]
allright it seems the feature only usefull for me.
closed

**Comment by burzum** [comment]
@kodok-buncit you can get this done much more easy with http://php.net/manual/en/function.glob.php than adding all of that code. Check the examples and comments there.

**Pull request closed without merging**